pkg/exchange/bitget/convert.go

```diff
@@ -29,6 +29,21 @@ func toGlobalMarket(s v2.Symbol) types.Market {
 		log.Warnf("The market symbol status %s is not online: %s", s.Symbol, s.Status)
 	}
 
+	var minQuantity, maxQuantity fixedpoint.Value
+	if v, err := fixedpoint.NewFromString(s.MinTradeAmount); err == nil {
+		minQuantity = v
+	} else {
+		log.Warnf("fail to convert min trade amount %s for %s, setting it to 0: %v", s.MinTradeAmount, s.Symbol, err)
+		minQuantity = fixedpoint.NewFromFloat(0.0)
```

Comment:
Maybe use the min fixedpoint like "1e-8"?

Reply:
Considering trading, I think `0` is a safer value to use as min amount.